rclcpp_examples/CMakeLists.txt

```diff
@@ -101,23 +101,36 @@ if(AMENT_ENABLE_TESTING)
     find_package("${middleware_impl}" REQUIRED)
   endforeach()
 
-  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters)
+  # Add each test case.  Multi-executable tests can be specified in
+  # colon-separated strings, like  exe1:exe2.
+  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters "talker;listener")
+
+    set(RCLCPP_EXAMPLES_EXPECTED_OUTPUT "")
+    foreach(executable ${example_executable})
+      set(RCLCPP_EXAMPLES_EXPECTED_OUTPUT "${RCLCPP_EXAMPLES_EXPECTED_OUTPUT} ${CMAKE_CURRENT_SOURCE_DIR}/test/${executable}")
```

Comment:
Why did you change it to space separators again? It simply doesn't work if you have spaces in the path.

After changing it back to a CMake list you might want to use `list(APPEND ...)` instead.